app/javascript/components/admin/AdminTable.js

```diff
@@ -423,6 +424,24 @@ class AdminTable extends React.Component {
     );
   };
 
+  /**
+   * Called when unlocked/locked status buttoned are toggled.
+   * Updates state and then calls table update handler.
+   * @param String status - user status value (locked or unlocked)
+   */
+  handleStatusChange = status => {
+    this.setState(
+      state => {
+        return {
+          query: { ...state.query, status },
```

Comment:
`page` should also be reset to the first page here. That way, the user will always be navigated to the first page when switching between filters.

```suggestion
          query: { ...state.query, status, page: 0 },
```

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/756/commits/1dae0d8e0faae7adde3a5f6ef8ba000b75383bfd

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/756/commits/880b8305535fbbb847be27d62ebe02cb77d0988f

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/756/commits/5a76982e17cecf1ae9ef903e658d846111e67ed8